onnxruntime/core/providers/openvino/ov_interface.cc

```diff
@@ -58,7 +58,7 @@ std::shared_ptr<OVNetwork> OVCore::ReadModel(const std::string& model, const std
     ov::AnyVector params{&modelStream, model_path};
 
     FE = manager.load_by_model(params);
-    if(FE) {
+    if (FE) {
```

Comment:
Have you checked the FE object for debug build is not throwing warning as error while building in Debug mode without below flag
![image](https://github.com/user-attachments/assets/fbda96e7-f62d-4813-9956-25c8fd95e5b9)

Refer below PR for insight- 
https://github.com/intel/onnxruntime/pull/383

Reply:
The issue with Debug build was unreachable Return statement which got fixed in the earlier PR. 
Refer to [https://github.com/intel/onnxruntime/commit/e85605258de8a53fda8532714aa23d2ac5b4b6c7#diff-6465519cbb5637fb777b39c7431bd86288284b5047d000cae74d5d7b36a308e5](https://github.com/intel/onnxruntime/commit/e85605258de8a53fda8532714aa23d2ac5b4b6c7#diff-6465519cbb5637fb777b39c7431bd86288284b5047d000cae74d5d7b36a308e5)
The change above is reflecting part of lint fix.